util/helm/client.go

```diff
@@ -106,7 +110,9 @@ func (c *nativeHelmChart) ExtractChart(chart string, version *semver.Version) (s
 	}
 	if !exists {
 		// always use Helm V3 since we don't have chart content to determine correct Helm version
-		helmCmd, err := NewCmdWithVersion(c.repoPath, HelmV3)
```

Comment:
nit: is `helmCmd := &Cmd{}` required. Can you please use `helmCmd, err := NewCmdWithVersion(c.repoPath, HelmV3, c.enableOci)`?